index.html

```diff
@@ -88,49 +76,121 @@
 <script>
   var data = [];
-  var options = {
+    var options = {
     yearRange: 'all'
   };
+  var svg, x, y, xAxis, yAxis; 
+  var margin = { top: 15, right: 15, bottom: 30, left: 45 };
+  var width = 640 - margin.right - margin.left;
+  var height = 400 - margin.top - margin.bottom;
+  var xpadding = 35;
+  var ypadding = 10;
 
-  var svg;
 
   // FETCH DATA
-
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
   function setup() {
+
     // SVG
 
-    svg = d3.select('#chart')
-      .append('svg');
 
+    svg = d3.select('#chart')
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.employment; }))
+      .range([height, 0])
+      .nice();
+    
     // AXES
+    //Define X axis
+      xAxis = d3.svg.axis()
+                .scale(x)
+                .orient("bottom")
+                .ticks(5);
+
+    //Define Y axis
+      yAxis = d3.svg.axis()
+                .scale(y)
+                .orient("left")
+                .ticks(5);
 
     // EVENT HANDLERS
-
     d3.select('#year-range').on('change', function () {
       options.yearRange = d3.event.target.value;
       update();
     });
   }
-
   function update() {
     // TRANSFORM
     txData = data.slice();
-
     if (options.yearRange !== 'all') {
       var yearRange = options.yearRange.split('-');
       txData = txData.filter(function (d) {
         return d.year >= yearRange[0] && d.year <= yearRange[1];
       });
     }
-
     // SCATTERPLOT
+    var points = svg.selectAll('.point')
+      .data(txData);
+    points.enter().append('circle')
+      .attr('class', 'point')
+      .attr('r', 5);
+    points
+      .attr('cx', function (d) { return x(d.unemployment); })
+      .attr('cy', function (d) { return y(d.employment); });
+    points.exit().remove();
+    
+    //Create labels
+      svg.selectAll("text")
+         .data(data)
+         .enter()
+         .append("text")
+         .text(function(d) {
+            return d[0] + "," + d[1];
+         })
+         .attr("x", function(d) {
+            return x(d[0]);
+         })
+         .attr("y", function(d) {
+            return y(d[1]);
+         })
+         .attr("font-family", "sans-serif")
+         .attr("font-size", "11px")
+         .attr("fill", "red");
+    
+        // Creating (printing the Axxis)
+    //Create X axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(0," + (height - xpadding) + ")")
```

Comment:
So if we were to offset it by just `height`, it would be shifted from the top of the inner box (the `g` element we created and offset on lines 106 and 107) to the bottom. But we subtract `xpadding` from that amount, which shifts it back up by that amount. It remains accurate, as we haven't shifted it left or right, but now awkwardly overlaps our data.